DigitalOffice/Services/ProjectService/Startup.cs

```diff
@@ -0,0 +1,62 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using ProjectService.Database;
+
+namespace ProjectService
+{
+    public class Startup
+    {   
+        public IConfiguration Configuration { get; }
+
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddDbContext<ProjectServiceDbContext>(options =>
+            {
+                options.UseSqlServer(Configuration.GetConnectionString("SQLConnectionString"));
+            });
+
+            services.AddControllers();
+        }
+        
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            UpdateDatabase(app);
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+            app.UseHttpsRedirection();
+
+            app.UseRouting();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapControllers();
+            });
+        }
+
+        private static void UpdateDatabase(IApplicationBuilder app)
```

Comment:
Why static btw?

Reply:
You can explain why this shouldn't be static?

As StackOverflow states, "static" allows the compiler to skip implicit "this." pointer and we make sure that this method can't change Startup's object state, etc.

Also this method will be called each time we start the application, so even a tiny performance gain would be good, I guess.

Reply:
> You can explain why this shouldn't be static?
> 
> As StackOverflow states, "static" allows the compiler to skip implicit "this." pointer and we make sure that this method can't change Startup's object state, etc.
> 
> Also this method will be called each time we start the application, so even a tiny performance gain would be good, I guess.

I didn't suggest any changes, just asked ;)